### PR TITLE
Feat : Private Route

### PR DESCRIPTION
**Pull request by anuj-kumary**
Added Private Route

Preview:-

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -2,22 +2,36 @@ import { Link } from 'react-router-dom';
...
   const filterHandler = () => {
     setShowFilter(true);
   };
 
+  const logOutHandler = (e) => {
+    e.preventDefault();
+    localStorage.removeItem('login');
+    setToken(null);
+    setUser(null);
+    // dispatch({
```
Comment (Rohanmond) [inline src/components/Navbar/Navbar.jsx:22]: remove unwanted comments

File: src/context/Data/data-context.js [hunk 2]
```diff
@@ -1,22 +1,38 @@
...
 
-  const noteResp = getNotes({ encodedToken: token });
-  if (noteResp.status === 200 || noteResp.status === 201) {
-    dispatch({
-      type: 'ADD_NOTE',
-      payload: { noteList: noteResp.data.notes },
-    });
-    console.log(noteResp.data.notes);
-  }
+  // const noteResp = getNotes({ encodedToken: token });
```
Comment (Rohanmond) [inline src/context/Data/data-context.js:29]: remove these comments

File: src/context/auth/auth-context.js [hunk 3]
```diff
@@ -8,10 +8,15 @@ const AuthContext = createContext();
 const AuthProvider = ({ children }) => {
   const localStorageToken = JSON.parse(localStorage.getItem('login'));
   const [token, setToken] = useState(localStorageToken?.token);
-  const localStorageUser = JSON.parse(localStorage.getItem('user'));
+  const localStorageUser = JSON.parse(localStorage.getItem('login'));
   const [user, setUser] = useState(localStorageUser?.user);
   const navigate = useNavigate();
 
+  // useEffect(() => {
```
Comment (Rohanmond) [inline src/context/auth/auth-context.js:15]: remove these also

**Review by Rohanmond — APPROVED** [review]

**Review by romabulani — APPROVED** [review]
As Rohan has already mentioned, remove unwanted comments. Rest all looks good to merge

File: src/context/Data/data-context.js [hunk 4]
```diff
@@ -1,22 +1,38 @@
...
+  // if (noteResp.status === 200 || noteResp.status === 201) {
+  //   dispatch({
+  //     type: 'ADD_NOTE',
+  //     payload: { noteList: noteResp.data.notes },
```
Comment (varnita619) [inline src/context/Data/data-context.js:33]: remove unwanted comments

**Review by varnita619 — APPROVED** [review]
Rest all looks good

File: src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (anuj-kumary) [reply src/components/Navbar/Navbar.jsx:22]: Done

File: src/context/auth/auth-context.js (hunk 3 above)
Comment (anuj-kumary) [reply src/context/auth/auth-context.js:15]: Done

File: src/context/Data/data-context.js (hunk 4 above)
Comment (anuj-kumary) [reply src/context/Data/data-context.js:33]: Done

File: src/context/Data/data-context.js (hunk 2 above)
Comment (anuj-kumary) [reply src/context/Data/data-context.js:29]: Done

**Pull request merged**